test/PhpSessionPersistenceTest.php

```diff
@@ -46,11 +46,46 @@ class PhpSessionPersistenceTest extends TestCase
      */
     private $persistence;
 
+    /**
+     * @var array
+     */
+    private $originalSessionSettings;
+
+    /**
+     * @var string
+     */
+    private $sessionSavePath;
+
     public function setUp()
     {
+        $this->sessionSavePath = sys_get_temp_dir() . "/zend-expressive-session-ext";
```

Comment:
Please use `mezzio` instead of `zend-expressive`.

Reply:
Fixed